src/scicookie/{{cookiecutter.project_slug}}/build-system/hatch-pyproject.toml

```diff
@@ -0,0 +1,119 @@
+[build-system]
+requires = ["hatchling>=1.17.1"]
+build-backend = "hatchling.build"
+
+[project]
+name = "{{ cookiecutter.project_name }}"
+authors = [
+  { name = "{{ cookiecutter.author_full_name }}", email = "{{ cookiecutter.author_email }}" },
+]
+description = "{{ cookiecutter.project_short_description }}"
+readme = "README.md"
+classifiers = [
+  "Development Status :: 1 - Planning",
+  "Intended Audience :: Science/Research",
+  "Intended Audience :: Developers",
+{%- if cookiecutter.project_license == "MIT" %}
+  "License :: OSI Approved :: MIT License",
+{%- elif cookiecutter.project_license == "BSD 3 Clause" %}
+  "License :: OSI Approved :: BSD License",
+{%- elif cookiecutter.project_license == "Apache Software License 2.0" %}
+  "License :: OSI Approved :: Apache Software License",
+{%- elif cookiecutter.project_license == "GNU General Public License v3" %}
+  "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+{%- endif %}
+  "Operating System :: OS Independent",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Topic :: Scientific/Engineering",
+  "Typing :: Typed",
+]
+dynamic = ["version"]
+requires-python = ">=3.8.1"
+dependencies = [
+{%- if cookiecutter.use_pytest == "yes" -%}
+    "pytest >= 7.3.2",
+{% if cookiecutter.use_coverage == "yes" -%}
+    "pytest-cov >= 4.1.0",
+{% endif %}
+{%- endif -%}{#- end of use_pytest -#}
+{%- if cookiecutter.use_hypothesis == "yes" -%}
+hypothesis = "^6.0"
```

Comment:
```suggestion
    hypothesis = "^6.0"
```

Reply:
not sure .. but I guess that hatch doesn't work with `^`
so , `^6.0` should be `>=6.0,<7.0`

Reply:
Oh yes, I completely missed that. That looks poetry.